src/main/java/org/jenkinsci/test/acceptance/plugins/plot/CsvDataSeries.java

```diff
@@ -0,0 +1,50 @@
+package org.jenkinsci.test.acceptance.plugins.plot;
+
+import org.jenkinsci.test.acceptance.po.Describable;
+import org.jenkinsci.test.acceptance.po.PageArea;
+
+/**
+ * Created by lphex on 4/6/17.
+ */
+@Describable("csv")
+public class CsvDataSeries extends DataSeries {
+    public CsvDataSeries(PageArea area, String path) {
+        super(area,  path );
+        this.selectType();
+    }
+
+
+    public void setUrl(String url){
+        control("url").set(url);
+    }
+
+    public  void setExclusionValues(String exclusionValues){
+        control(getFileType() + "/exclusionValues").set(exclusionValues);
+    }
+
+    public void checkDispalyTabel(){
+        control(getFileType() + "/displayTableFlag").check();
+    }
+
+    public void selectIncludeAllColumns(){
```

Comment:
Space between ) and {, see [coding style](https://github.com/uhafner/codingstyle/blob/master/doc/Formatierung.md).